cmd/geth/accountcmd_test.go

```diff
@@ -88,6 +88,38 @@ Path of the secret key file: .*UTC--.+--[0-9a-f]{40}
 `)
 }
 
+func TestAccountImport(t *testing.T) {
+	bytes64 := "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+	success := `Address: {[0-9a-f]{40}}`
+	failure := `Fatal: Failed to load the private key: expected 64 bytes, got \d+`
+	keyToMsg := make(map[string]string)
+	keyToMsg[bytes64] = success
+	keyToMsg[bytes64[:40]] = failure
+	keyToMsg[bytes64+"\n"] = success
+	keyToMsg[bytes64+"\r\n"] = success
+	keyToMsg[bytes64+"1"] = failure
+	keyToMsg[bytes64+"x"] = failure
+	keyToMsg[bytes64+bytes64] = failure
```

Comment:
The canonical way to define table driven tests would be:

```go
tests := []struct{
  key string
  err error
} {
  {key: "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef", err: nil},
  {key: "0123456789abcdef0123456789abcdef01234567", err: errSomethingSomething},
  [...]
}
```

Please also define a separate error constant if you want to check for it. Using a string is suuuuper brittle. Alternatively wait until Go 1.14.1 is out so we can deprecate 1.11 and 1.12 and then use the fancy new error constructs.

Reply:
Also please don't use regexps for error or success checks. Since you know the exact input, you also know the exact output. Using regexps would just hide potential errors.

Reply:
- I refactored to canonical table-driven test format
- I replaced regexps with exact matches

I'm still using the `bytes64 + "\n"` format because I find it more understandable than a long string with a `\n` at the end. I can change that, too.

Since I compare the string returned by the command line, I don't know how to use an error here. The underlying `TestCmd` doesn't expose the error returned by the comparison, it simply calls `t.Fatal()` if there's a mismatch.

Reply:
> I'm still using the bytes64 + "\n" format because I find it more understandable than a long string with a \n at the end. I can change that, too.

Please change it, this is a short test suite, but when things get bigger (e.g. hundreds of lines of test cases), you don't want to keep jumping back and forth to find what variables in there mean.

Reply:
Ok, replaced with string literals